### ISSUE TITLE
[Accessibility] Participants in XMPP conferences not accessible to NVDA screen reader

### ISSUE THREAD

**Issue by MarcoZehe** [miranda-ng/miranda-ng#1589]
### Expected behavior
Ability to hear the names of other members in an XMPP group chat.

### Actual behavior
NVDA only reads "entry" for each item.

### Does it work in stable version?
No. Both 0.95.8 and current 0.95.9 are affected.

### Steps to reproduce that behavior
1. Start Miranda NG and the NVDA screen reader.
2. Join or open an XMPP group chat.
3. Shift-tab several times to go to the list of participants.
4. Arrow up and down.

NVDA version 2018.2.1, Miranda NG 0.95.9. Windows 10 17134.

When installing Miranda NG, I chose the "list for blind users" option, and the contact list works fine. Only the list of participants in the XMPP group chat doesn't.

**Comment by Piciok** [comment]
Hi Marco!

Which chat window plugin did you choose?

I found this works best with Scriver and is fine here on development versions.

Wysłane z aplikacji Poczta dla Windows 10

Od: Marco Zehe
Wysłano: sobota, 8 września 2018 14:47
Do: miranda-ng/miranda-ng
DW: Subscribed
Temat: [miranda-ng/miranda-ng] [Accessibility] Participants in XMPPconferences not accessible to NVDA screen reader (#1589)

Expected behavior
Ability to hear the names of other members in an XMPP group chat.
Actual behavior
NVDA only reads "entry" for each item.
Does it work in stable version?
No. Both 0.95.8 and current 0.95.9 are affected.
Steps to reproduce that behavior
1. Start Miranda NG and the NVDA screen reader.
2. Join or open an XMPP group chat.
3. Shift-tab several times to go to the list of participants.
4. Arrow up and down.
NVDA version 2018.2.1, Miranda NG 0.95.9. Windows 10 17134.
When installing Miranda NG, I chose the "list for blind users" option, and the contact list works fine. Only the list of participants in the XMPP group chat doesn't.
